cfme/tests/configure/test_access_control.py

```diff
@@ -803,16 +804,17 @@ def test_superadmin_tenant_crud(request):
         * Update name of tenat
         * Delete tenant
     """
-    tenant = Tenant(
+    tc = TenantCollection(appliance=appliance)
+    tenant = tc.create(
         name='tenant1' + fauxfactory.gen_alphanumeric(),
-        description='tenant1 description')
+        description='tenant1 description',
+        parent=tc.get_root_tenant())
```

Comment:
**QUESTION**: This doesn't look right to me structurally. I would expect something like this:

```python
tc = TenantCollection(...)
rt = tc.root_tenant()
t = rt.tenants.create(...)   # without parent
# or in case of projects under tenant
p = rt.projects.create(...)
```

Your opinion here @psav @mshriver ?

Reply:
Ah yes, this looks better way to do, changing